platforms/core-execution/build-cache-base/src/main/java/org/gradle/caching/internal/origin/OriginMetadataFactory.java

```diff
@@ -29,28 +29,16 @@ public class OriginMetadataFactory {
     private static final String CACHE_KEY = "buildCacheKey";
     private static final String CREATION_TIME_KEY = "creationTime";
     private static final String EXECUTION_TIME_KEY = "executionTime";
-    private static final String OPERATING_SYSTEM_KEY = "operatingSystem";
-    private static final String HOST_NAME_KEY = "hostName";
```

Comment:
🤔 The host name was often useful when we didn't have a scan for the build that produced the cache entry. Without it, it will basically be impossible to figure that out.